providers/src/airflow/providers/edge/worker_api/routes/_v2_routes.py

```diff
@@ -81,18 +80,12 @@ def _initialize_method_map() -> dict[str, Callable]:
     from airflow.providers.edge.models.edge_logs import EdgeLogs
     from airflow.providers.edge.models.edge_worker import EdgeWorker
     from airflow.secrets.metastore import MetastoreBackend
-    from airflow.sensors.base import _orig_start_date
-    from airflow.utils.cli_action_loggers import _default_action_log_internal
-    from airflow.utils.log.file_task_handler import FileTaskHandler
 
     functions: list[Callable] = [
-        _default_action_log_internal,
```

Comment:
This will break the edge worker. As comments above this is atm an intentional copy from Airflow 2.10 interface state.
For mypy to pass you need to add a type ignore (like with the other methods which are gone in parallel or wait until I have merged #44434 which just removes the pieces of code.

Reply:
Now the PR is merged, so probably after re-base you can drop these changes and should not be in-the-way anymore